docs/posts/bigquery-arrays/index.qmd

```diff
@@ -0,0 +1,247 @@
+---
+title: Working with arrays in Google BigQuery
+author: "Phillip Cloud"
+date: "2023-09-12"
+categories:
+    - release
```

Comment:
are you intending this as a "release blog?" -- I was thinking of the `release` category as a running series of blogs that cover any release with major-ish features more indepth than release notes

Reply:
Nope, thanks!